src/components/SEO.js

```diff
@@ -26,5 +26,11 @@ export default () => (
             content="FraserHacks 2020 coming soon! Hosted at John Fraser Secondary School"
         />
         <meta property="twitter:image" content="/banner.jpg" />
+        <script
```

Comment:
just extract the SVGs and load them in as JSX instead of including the entire library